crates/cargo-test-support/Cargo.toml

```diff
@@ -11,7 +11,7 @@ doctest = false
 cargo = { path = "../.." }
 cargo-test-macro = { path = "../cargo-test-macro" }
 filetime = "0.2"
-flate2 = "1.0"
+flate2 = { version = "1.0", default-features = false }
```

Comment:
I don't think it's worthwhile to change this since if this were the only dependency on `flate2` it wouldn't actually compile (the crate requires at least one feature to bet set).

Reply:
Not sure what you mean by that: I tested the change locally and the amount of compiled crates went down from IIRC 155 to 153 :thinking:

Reply:
It works because the features are unified with cargo (and picks up the "zlib" feature).  In isolation, the cargo-test-support crate wouldn't build (because flate2 needs at least one backend).  That won't happen in practice, but it would be probably be best to be explicit and list `features = ["zlib"]` here, too.

Reply:
Thank you for the clarification! I have updated Cargo.toml for test support to explicitly include that feature.